crates/live_update/src/jni.rs

```diff
@@ -53,7 +77,15 @@ fn jni_fetch_doc<'local>(
         }
     };
 
-    let ser_result = match jni_fetch_doc_impl(&mut env, doc_id, request_json) {
+    let proxy_config: ProxyConfig = match get_proxy_config(&mut env, &jproxy_config) {
+        Ok(it) => it,
+        Err(e) => {
+            eprintln!("Error getting proxy_config - {e}");
```

Comment:
Please use the rust logger macros for printing errors

```suggestion
            error!("Error getting proxy_config - {e}");
```

Reply:
Done.